app/models/state_file_base_intake.rb

```diff
@@ -372,20 +372,22 @@ def sanitize_bank_details
   end
 
   def primary_senior?
-    calculate_age(inclusive_of_jan_1: true, dob: primary_birth_date) >= 65
+    calculate_age(primary_birth_date, inclusive_of_jan_1: true) >= 65
   end
 
   def spouse_senior?
-    return nil unless spouse_birth_date.present?
+    return false unless spouse_birth_date.present?
 
-    calculate_age(inclusive_of_jan_1: true, dob: spouse_birth_date) >= 65
+    calculate_age(spouse_birth_date, inclusive_of_jan_1: true) >= 65
   end
 
-  def calculate_age(inclusive_of_jan_1: true, dob: primary_birth_date)
-    # federal guidelines: you qualify for age related benefits the day before your birthday
-    # that means for a given tax year those born on Jan 1st the following tax-year will be included
-    # this does not apply for benefits you age out of or any age calculations for Maryland
-    raise StandardError, "Primary or spouse missing date-of-birth" if dob.nil?
+  def calculate_age(dob, inclusive_of_jan_1:)
```

Comment:
I changed the order of arguments because the dob is more relevant to the method.

I also removed the default values because I think it is better to always provide them explicitly.